src/components/Learn-Components/Card-Component/learn-card.style.js

```diff
@@ -5,6 +5,13 @@ const CardWrapper = styled.div`
     min-height: 16rem;
     margin: auto;
     border-radius: 1rem;
+    cursor: pointer;
+    box-shadow: rgba(0, 179, 158, 0.9) 0px 2px 10px 1px;
```

Comment:
changes looks good, but the box shadow does not have the perfect curve effect at corners if you see
![image](https://github.com/layer5io/layer5/assets/90546692/b6336200-e55c-48dc-9150-7b8591f1b5ff)

Reply:
ok I'll look into it